packages/react-docsite-components/src/components/Markdown/Markdown.tsx

```diff
@@ -30,9 +30,12 @@ const getClassNames = classNamesFunction<IMarkdownStyleProps, IMarkdownStyles>()
 const MarkdownBase: React.FunctionComponent<IMarkdownProps> = props => {
   const { styles, theme, children } = props;
   const classNames = getClassNames(styles, { theme: theme! });
+
+  const content = (children as { default: React.ReactNode }).default ?? children;
```

Comment:
What happened here?

Reply:
something unexplainable... i think it related to bumping react typings but i'm not sure

Reply:
Ken's fault!

Actually there was a change in the rawloader in that it outputs esm by default; but the output changes then, so that rather than importing `* as Foo` where Foo becomes a string, it returns an object with a `default` prop.

Ken worked around it here by accepting either string or object with default; however there are other cases where we import using raw-loader and that workaround would have had to been applied in a number of proprietary places. 

So I opted for the safer option; set the config value in the raw loader imports to avoid the ESM output. This ensures that we have no accidental misses in updating the code, and is a much less error prone change.

Reply:
I am going to undo this as my change should be enough.